website_anchor_smooth_scroll/static/src/js/website_anchor_smooth_scroll.js

```diff
@@ -1,25 +1,29 @@
 /* © 2016 Antiun Ingeniería S.L. - Jairo Llopis
  * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+odoo.define('website_anchor_smooth_scroll.website_anchor_smooth_scroll', function (require) {
 
+require('web.dom_ready');
```

Comment:
Do not put anything before `"use strict"` or it has no meaning. Besides, this call must appear above 1st dom modifications, not here.